setup.cfg

```diff
@@ -42,7 +42,7 @@ install_requires =
     # pyproj 2.2+ required for CRS object
     pyproj>=2.2,<4
     # pytorch-lightning 1.5.1+ required for apply_to_collection bugfix
-    pytorch-lightning>=1.5.1,<2
+    pytorch-lightning[extra]>=1.5.1,<2
```

Comment:
This is painful. Lightning dropped tensorboardX as a required dep. TorchGeo technically doesn't require tensorboardX, but train.py does. Some of our trainers may or may not be broken when plotting if tensorboard isn't installed. So technically we could just add a dep on tensorboard, even though we don't import it. The alternative is to depend on `pytorch-lightning[extra]` which adds tensorboardX and a bunch of other deps we don't need. I don't love either solution.